src/java.desktop/share/classes/sun/print/CustomOutputBin.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
```
The year shouldn't be replaced but added.

Reply:
I keep forgetting that... Thanks a lot!